src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -77,10 +77,6 @@ public class WebScriptHostManager : ScriptHostManager
             }
 
             config.IsSelfHost = webHostSettings.IsSelfHost;
-            if (config.HostConfig.LoggerFactory == null)
```

Comment:
We're already defaulting this in ScriptHost, so it's confusing why we'd be doing this in two places.